D/DynarePreprocessor/build_tarballs.jl

```diff
@@ -0,0 +1,44 @@
+using BinaryBuilder
+
+
+name = "DynarePreprocessor"
+version = v"4.8.0"
+sources = [
+    GitSource("https://git.dynare.org/Dynare/preprocessor.git", "a8fce06dc46ca09329e6899e1fde47f2cd81809b"),
+    DirectorySource("./bundled")
```

Comment:
Looks like you didn't commit your patch file 😉

Reply:
:)